src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -42,7 +42,7 @@
     <PackageReference Include="Microsoft.Azure.Functions.NodeJsWorker" Version="1.1.1" />
     <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker" Version="1.0.194" />
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.14" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions" Version="3.0.4" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions" Version="3.0.5-10773" />
```

Comment:
This is pulling in the timer log updates that went in with https://github.com/Azure/azure-webjobs-sdk-extensions/pull/599